<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginNetSetWindow</class>
 <widget class="QWidget" name="LoginNetSetWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>334</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>428</width>
    <height>334</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>428</width>
    <height>334</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LoginNetSetWindow</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>68</y>
     <width>52</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>网络设置</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>108</y>
     <width>29</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>类型:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxNetType">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>106</y>
     <width>120</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{font-family:Microsoft YaHei;}
/*下拉列表框*/
QComboBox
{
    background:white;
    padding-left:2px ;
    border-radius:2px;
    border: 1px solid rgb(173 , 173 , 173);
}
QComboBox:hover
{
    border: 1px solid rgb(21 , 131 , 221);
}
QComboBox QAbstractItemView::item
{
    height:24px;
}
QComboBox::down-arrow
{
    border-image:url(:/Resources/LoginWindow/drop_down_Button.png) 0 17 0 17 ;
}

QComboBox::down-arrow:on
{
    border-image:url(:/Resources/LoginWindow/drop_down_Button.png) 0 0 0 34 ;
}
QComboBox::drop-down
{
    width:20px;
    background:transparent; /*不加此句下拉箭头背景色为灰色与整体样式不一致，也可设置 border:0px; border-radius:0px; background:white; border-left:0px ; 即设置为无边框*/
    padding-right:5px;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>192</x>
     <y>108</y>
     <width>29</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>地址:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>321</x>
     <y>108</y>
     <width>29</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>端口:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>138</y>
     <width>42</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>用户名:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>192</x>
     <y>138</y>
     <width>29</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>密码:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>333</x>
     <y>138</y>
     <width>16</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>域:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>196</y>
     <width>65</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>登录服务器</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>234</y>
     <width>29</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>类型:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>192</x>
     <y>234</y>
     <width>29</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>地址:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>321</x>
     <y>234</y>
     <width>29</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>端口:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxServerType">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>233</y>
     <width>120</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>24</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">*{font-family:Microsoft YaHei;}
/*下拉列表框*/
QComboBox
{
    background:white;
    padding-left:2px ;
    border-radius:2px;
    border: 1px solid rgb(173 , 173 , 173);
}
QComboBox:hover
{
    border: 1px solid rgb(21 , 131 , 221);
}
QComboBox QAbstractItemView::item
{
    height:24px;
}
QComboBox::down-arrow
{
    border-image:url(:/Resources/LoginWindow/drop_down_Button.png) 0 17 0 17 ;
}

QComboBox::down-arrow:on
{
    border-image:url(:/Resources/LoginWindow/drop_down_Button.png) 0 0 0 34 ;
}
QComboBox::drop-down
{
    width:20px;
    background:transparent; /*不加此句下拉箭头背景色为灰色与整体样式不一致，也可设置 border:0px; border-radius:0px; background:white; border-left:0px ; 即设置为无边框*/
    padding-right:5px;
}</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_3">
   <property name="geometry">
    <rect>
     <x>224</x>
     <y>233</y>
     <width>88</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>88</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>88</width>
     <height>24</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">/*下拉列表框*/
QComboBox
{
    background:white;
    padding-left:2px ;
    border-radius:2px;
    border: 1px solid rgb(173 , 173 , 173);
}
QComboBox:hover
{
    border: 1px solid rgb(21 , 131 , 221);
}
QComboBox QAbstractItemView::item
{
    height:24px;
}
QComboBox::down-arrow
{
    border-image:url(:/Resources/LoginWindow/drop_down_Button.png) 0 17 0 17 ;
}

QComboBox::down-arrow:on
{
    border-image:url(:/Resources/LoginWindow/drop_down_Button.png) 0 0 0 34 ;
}
QComboBox::drop-down
{
    width:20px;
    background:transparent; /*不加此句下拉箭头背景色为灰色与整体样式不一致，也可设置 border:0px; border-radius:0px; background:white; border-left:0px ; 即设置为无边框*/
    padding-right:5px;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>224</x>
     <y>106</y>
     <width>88</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>88</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>88</width>
     <height>24</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
    background:white;
    padding-left:2px ;
    border-radius:2px;
    border: 1px solid rgb(173 , 173 , 173);
}
QLineEdit:hover
{
    border: 1px solid rgb(21 , 131 , 221);
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>353</x>
     <y>106</y>
     <width>56</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>56</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>56</width>
     <height>24</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
    background:white;
    padding-left:2px ;
    border-radius:2px;
    border: 1px solid rgb(173 , 173 , 173);
}
QLineEdit:hover
{
    border: 1px solid rgb(21 , 131 , 221);
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>224</x>
     <y>138</y>
     <width>88</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>88</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>88</width>
     <height>24</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
    background:white;
    padding-left:2px ;
    border-radius:2px;
    border: 1px solid rgb(173 , 173 , 173);
}
QLineEdit:hover
{
    border: 1px solid rgb(21 , 131 , 221);
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>138</y>
     <width>120</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>24</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
    background:white;
    padding-left:2px ;
    border-radius:2px;
    border: 1px solid rgb(173 , 173 , 173);
}
QLineEdit:hover
{
    border: 1px solid rgb(21 , 131 , 221);
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>353</x>
     <y>138</y>
     <width>56</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>56</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>56</width>
     <height>24</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
    background:white;
    padding-left:2px ;
    border-radius:2px;
    border: 1px solid rgb(173 , 173 , 173);
}
QLineEdit:hover
{
    border: 1px solid rgb(21 , 131 , 221);
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_6">
   <property name="geometry">
    <rect>
     <x>353</x>
     <y>233</y>
     <width>56</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>56</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>56</width>
     <height>24</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
    background:white;
    padding-left:2px ;
    border-radius:2px;
    border: 1px solid rgb(173 , 173 , 173);
}
QLineEdit:hover
{
    border: 1px solid rgb(21 , 131 , 221);
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButtonTestNetWork">
   <property name="geometry">
    <rect>
     <x>353</x>
     <y>168</y>
     <width>56</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>56</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>56</width>
     <height>24</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
    background:rgb(245,249,252);
    border-radius:2px;
    border: 1px solid rgb(173 , 173 , 173);
}
QPushButton:pressed
{
    background:rgb(245,249,252);
    border-radius:2px;
	padding-left:2px;
	padding-top:2px;
    border: 1px solid rgb(173 , 173 , 173);
}

</string>
   </property>
   <property name="text">
    <string>测试</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButtonOk">
   <property name="geometry">
    <rect>
     <x>247</x>
     <y>305</y>
     <width>84</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
    background:rgb(245,249,252);
    border-radius:2px;
    border: 1px solid rgb(173 , 173 , 173);
}
QPushButton:pressed
{
    background:rgb(245,249,252);
    border-radius:2px;
	padding-left:2px;
	padding-top:2px;
    border: 1px solid rgb(173 , 173 , 173);
}

</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButtonCancel">
   <property name="geometry">
    <rect>
     <x>335</x>
     <y>305</y>
     <width>84</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
    background:rgb(245,249,252);
    border-radius:2px;
    border: 1px solid rgb(173 , 173 , 173);
}
QPushButton:pressed
{
    background:rgb(245,249,252);
    border-radius:2px;
	padding-left:2px;
	padding-top:2px;
    border: 1px solid rgb(173 , 173 , 173);
}

</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
